plugins/scripts/inject-config.js

```diff
@@ -83,8 +83,44 @@ module.exports = function injectOptimoveConfig(context) {
         console.info('Optimove: Preparing Android platform...');
         prepareAndroid(context, optimoveConfig);
     }
+
+    if (hasPlatform(context, 'ios')) {
+        console.info('Optimove: Preparing iOS platform...');
+        prepareIos(context, optimoveConfig);
+    }
 };
 
+function prepareIos(context, OptimoveConfig) {
+    const iosPath = path.join(context.opts.projectRoot, 'platforms', 'ios');
+    const files = fs.readdirSync(iosPath);
+
+    const xcodeProj = files.find(name => name.indexOf('.xcodeproj') > -1);
+    const targetName = xcodeProj.replace('.xcodeproj', '');
+
+    const configDest = path.join(
+        iosPath,
+        targetName,
+        'Resources',
+        'optimove.plist'
+    );
+
+    if (!fs.existsSync(configDest)) {
+        console.error(
+            'optimove: optimove.plist resource not found, aborting setup'
+        );
+        return;
+    }
+
+    const config = renderTemplate('optimove.plist', {
+        API_KEY: OptimoveConfig.apiKey,
+        SECRET_KEY: OptimoveConfig.secretKey,
+        ENABLE_CRASH: OptimoveConfig.enableCrashReporting,
+        IN_APP_STRATEGY: OptimoveConfig.inAppConsentStrategy
```

Comment:
I think we need to align the keys in the template plist, the inject-config script, and the swift code?

There should be two plist keys, one for optimove credentials, and another for mobile credentials, similar to Android.

Reply:
done